mmv1/third_party/terraform/services/compute/resource_compute_instance_template.go.erb

```diff
@@ -56,6 +56,7 @@ func ResourceComputeInstanceTemplate() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceComputeInstanceTemplateCreate,
 		Read:   resourceComputeInstanceTemplateRead,
+		Update: resourceComputeInstanceTemplateRead,
```

Comment:
Oh, wait- we want to verify every subfield is `ForceNew`. Some will inadvertently become updatable now.

Reply:
good catch.